src/pages/index.page.tsx

```diff
@@ -141,6 +142,21 @@ const Home = () => {
               </AlertDialog>
             </div>
           </div>
+
+          <div className={styles.card}>
+            <h2>:: Flash Toast</h2>
+            <div>
+              <FlashToast
+                title="Error"
+                messageList={[
+                  'lorem ipsum dolor sit amet',
+                  'consectetur adipiscing elit',
+                  'integer molestie lorem at massa',
+                  'facilisis in pretium nisl aliquet',
+                ]}
```

Comment:
I am just wondering do we need to set the dummy string here? 🤔

Reply:
On this page, I use it as a styleguide page.
it's just some require text for rendering the component.